### PR TITLE
[V3] Add prompt if message deletion > 100

### PR DESCRIPTION
**Pull request by Bakersbakebread**
As per issue #1295. 

Add's the prompt "Are you sure you want to delete {number} messages? (y/n)"

### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

add's a prompt to user if messages to be deleted > 100.

prompt is:
Are you sure you want to delete {number} messages? (y/n)

### CODE REVIEW

**Review by palmtree5 — APPROVED** [review]

**Comment by mikeshardmind** [comment]
This needs a check inside the wait for message that they author is the person the bot got the message for, channel should probably be included in that check as well

**Review by palmtree5 — CHANGES_REQUESTED** [review]
Needs a check to only accept answers from the user running the command

**Comment by Bakersbakebread** [comment]
Checks added, also deleted the "Continuing" message for maximum deletion.

Deleteception.

File: redbot/cogs/cleanup/cleanup.py [hunk 1]
```diff
@@ -41,6 +41,20 @@ def __init__(self, bot: Red):
...
+        if number > 100:
+            await ctx.send('Are you sure you want to delete {} messages? (y/n)'.format(number))
+            response = await self.bot.wait_for('message', check=author_check)
+
+            if response.content.startswith('y'):
+                tmp = await ctx.send('Continuing..')
+                await tmp.delete()
+            else:
+                return await ctx.send('Cancelled.')
+        await tmp.delete()
```
Comment (palmtree5) [inline redbot/cogs/cleanup/cleanup.py]: Unneeded?

**Review by palmtree5 — CHANGES_REQUESTED** [review]

File: redbot/cogs/cleanup/cleanup.py (hunk 1 above)
Comment (Bakersbakebread) [reply redbot/cogs/cleanup/cleanup.py]: Definitely.

**Review by palmtree5 — APPROVED** [review]

**Pull request merged**